src/frontend/apps/drive/src/features/explorer/components/modals/move/ExplorerMoveFolder.tsx

```diff
@@ -0,0 +1,389 @@
+    ) {
+      return;
+    }
+
+    if (itemId && item === undefined) {
+      return;
+    }
+    const data = getMoveData();
+    if (data.newRootId !== data.oldRootParentId) {
+      moveConfirmationModal.open();
+      return;
+    }
+
+    handleMove(data.ids, data.newParentId, data.oldParentId);
+  };
+
+  return (
+    <>
+      <Modal
+        isOpen={isOpen}
+        closeOnClickOutside
+        title={
+          <div className="modal__move__header">
+            <span className="modal__move__title">Déplacer</span>
+            <span className="modal__move__description">
+              <Trans
+                i18nKey={
+                  itemsToMove.length === 1
+                    ? "explorer.modal.move.description_one_item"
+                    : "explorer.modal.move.description_multiple_items"
+                }
+                values={{
+                  count: itemsToMove.length,
+                  name: itemsToMove[0].title,
+                }}
+              />
+            </span>
+          </div>
+        }
+        onClose={onCloseModal}
+        size={isDesktop ? ModalSize.LARGE : ModalSize.FULL}
+        rightActions={
+          <div className="modal__move__footer">
+            <Button color="secondary" onClick={onCloseModal}>
+              {t("common.cancel")}
+            </Button>
+            <Button
+              color="primary"
+              disabled={!itemId && selectedItems.length === 0}
+              onClick={onMove}
+            >
+              {t("explorer.modal.move.move_button")}
+            </Button>
+          </div>
+        }
+      >
+        <div className="noPadding">
+          <div className="modal__move">
+            <HorizontalSeparator />
+          </div>
+          <div
+            className="modal__move__content"
+            style={{ height: isDesktop ? "300px" : "100%" }}
+          >
+            <div
+              className="modal__move__content__container"
+              onClick={(e) => {
+                const target = e.target as HTMLElement;
+                const isList = target.closest(".c__datagrid__table__container");
+                const isBreadcrumb = target.closest(
+                  ".modal__move__breadcrumbs"
+                );
+                if (isList || isBreadcrumb) {
+                  return;
+                }
+                setSelectedItems([]);
+              }}
+            >
+              <div className="modal__move__breadcrumbs">
+                <Breadcrumbs items={breadcrumbsItems} />
+
+                {itemId && (
+                  <Button
+                    size="small"
+                    color="primary-text"
+                    onClick={createFolderModal.open}
+                  >
+                    <img src={add_folder.src} alt="add" />
+                  </Button>
+                )}
+              </div>
+              <div
+                className={clsx(
+                  "c__datagrid explorer__grid explorer__compact",
```

Comment:
I annoys me a bit that the style of the compact ExplorerGridItemsList is not hold by ExplorerGridItemsList itself, but by this parent div. 

If we want to reuse the ExplorerGridItemsList compact elsewhere in another compact it is not going to display properly, SRP issue?